daemon/containerd/image_snapshot.go

```diff
@@ -178,10 +178,10 @@ func (i *ImageService) GetLayerByID(cid string) (container.RWLayer, error) {
 	}
 
 	switch len(lss) {
-	case 0:
-		return nil, errdefs.NotFound(errors.New("rw layer lease not found for container " + cid))
 	default:
 		log.G(ctx).WithFields(log.Fields{"container": cid, "leases": lss}).Warn("multiple leases with the same id found, this should not happen")
```

Comment:
I think in most places, we use `default` as last one; could you move it to be the last one instead?